### PR TITLE
translations PR 20180625

### PR DESCRIPTION
**Pull request by enahum**
Ideally this PR should be review by community members for each language.

Note: In the event that you see an english translation instead of the desired language is because this PR was submitted before the translation was made on the Mattermost Translation Server and probably will be submitted for the next PR.

Webapp PR: https://github.com/mattermost/mattermost-webapp/pull/1378

### CODE REVIEW

**Comment by enahum** [comment]
@jasonblais you'll need to ask @jespino maybe caused by his recent work

**Comment by jespino** [comment]
@jasonblais Yes, exactly, this is the "propagation" of the previous PR where we remove unused strings from the en.json file. After that, @enahum upload the en.json to translate.mattermost.com, and download the update translations (which include the deletions of unused strings).

**Comment by enahum** [comment]
I'm ok merging this if that is what is expected, I still see a lot of blank strings in the translation server and I do believe that should be fixed ASAP

File: i18n/de.json [hunk 1]
```diff
@@ -1,51 +1,11 @@
 [
   {
-    "id": "April",
-    "translation": "April"
+    "id": "actiance.xml.output.formatter.marshalToXml.appError",
+    "translation": ""
   },
   {
-    "id": "August",
```
Comment (jasonblais) [inline i18n/de.json]: I'm not sure why we're removing the ability to translate months for instance?

**Comment by jasonblais** [comment]
I'm just confused why we're removing so many strings here, e.g. it looks like months will no longer be translateable

**Comment by jasonblais** [comment]
I'll prioritize reviews for https://github.com/mattermost/mattermost-server/pull/8987 so we can merge that today and fix the issue with blank strings

File: i18n/de.json (hunk 1 above)
Comment (enahum) [reply i18n/de.json]: I think it is because is no longer being used

Comment (jasonblais) [reply i18n/de.json]: Aren't they used?

image

Comment (jasonblais) [reply i18n/de.json]: Unless those are generated differently now. But I don't see a way to translate the months to different languages,

Comment (enahum) [reply i18n/de.json]: does are in the webapp not the server and those are generated with JS code that uses locale

Comment (jasonblais) [reply i18n/de.json]: Okay I see.

**Pull request merged**